app/components/UI/AccountFromToInfoCard/AccountFromToInfoCard.test.tsx

```diff
@@ -147,4 +166,59 @@ describe('AccountFromToInfoCard', () => {
     );
     expect(await findByText('0xF4e8...287B')).toBeDefined();
   });
+
+  describe('from account balance', () => {
+    const ERC20Transaction = {
+      assetType: 'ERC20',
+      data: '0xa9059cbb0000000000000000000000002f318c334780961fb129d2a6c30d0763d9a5c9700000000000000000000000000000000000000000000000000000000000003a98',
+      from: '0x1',
+      selectedAsset: {
+        address: '0x326836cc6cd09B5aa59B81A7F72F25FcC0136b95',
+        decimals: '4',
+        image: 'https://metamask.github.io/test-dapp/metamask-fox.svg',
+        isERC721: false,
+        symbol: 'TST',
+      },
+      to: '0x2f318c334780961fb129d2a6c30d0763d9a5c970',
+      transaction: {
+        data: '0xa9059cbb0000000000000000000000002f318c334780961fb129d2a6c30d0763d9a5c9700000000000000000000000000000000000000000000000000000000000003a98',
+        from: '0x1',
+        to: '0x2f318c334780961fb129d2a6c30d0763d9a5c970',
+        value: '3a98',
+      },
+    };
+    let mockGetERC20BalanceOf: any;
+    beforeEach(() => {
+      mockGetERC20BalanceOf = jest.fn().mockReturnValue(0x0186a0);
+      Engine.context.AssetsContractController = {
+        getERC20BalanceOf: mockGetERC20BalanceOf,
+      };
+    });
+
+    it('should render balance from AssetsContractController.getERC20BalanceOf if selectedAddress is different from fromAddress', async () => {
```

Comment:
```suggestion
    it('should render balance from AssetsContractController.getERC20BalanceOf if selectedAddress is different from fromAddress', () => {
```

no need for async for these tests